CanonicalLtd/maas-docs
# Add canonical links to link to 2.4

QA
--

`./run` and go to e.g. http://localhost:8203/2.2/en/installconfig-snap-install.

View source, and check for a canonical link to the `2.4` version of the document:

`<link rel="canonical" href="https://docs.maas.io/2.4/en/installconfig-snap-install" />`

Check the link contained there is accurate, and check some other pages for similar links.
